### PR TITLE
fix: tooltip flicker and sizing

### PR DESCRIPTION
**Pull request by mscolnick**

### CODE REVIEW

File: frontend/src/core/codemirror/completion/hints.ts [hunk 1]
```diff
@@ -116,11 +116,6 @@ const cursorTooltipField = StateField.define<Tooltip[]>({
     return [];
   },
   update(tooltips, tr) {
-    // If the document or selection has changed, clear the tooltips
-    if (tr.docChanged || tr.selection) {
-      return [];
-    }
-
```
Comment (akshayka) [inline frontend/src/core/codemirror/completion/hints.ts:123]: Maybe this is needed sometimes? I see completion overlapping with tooltip now. See video below

[multiple-completions-tooltips.webm](https://github.com/marimo-team/marimo/assets/1994308/29b1a328-507f-4b4d-8f44-92ae4a6a1d44)

Comment (mscolnick) [reply frontend/src/core/codemirror/completion/hints.ts:123]: i think i can clear this in a different place to avoid the overlap

**Review by akshayka — APPROVED** [review]
nice!

**Comment by akshayka** [comment]
@mscolnick Ah sorry, I found another issue post-merge:

image

autocomplete and hover showing at some time, hover covering completion

**Pull request merged**